ctms/app.py

```diff
@@ -625,25 +619,21 @@ def partial_update_ctms_contact(
 )
 def read_ctms_in_bulk_by_timestamps_and_limit(
     start: datetime,
-    end: Optional[Union[datetime, str]] = None,
-    limit: Optional[Union[int, str]] = None,
+    end: Optional[Union[datetime, Literal[""]]] = None,
+    limit: Optional[Union[int, Literal[""]]] = None,
     after: Optional[str] = None,
-    mofo_relevant: Optional[Union[bool, str]] = None,
+    mofo_relevant: Optional[Union[bool, Literal[""]]] = None,
     db: Session = Depends(get_db),
     api_client: ApiClientSchema = Depends(get_enabled_api_client),
 ):
-    after_param = updates_helper(value=after, default=None)
-    limit_param = updates_helper(value=limit, default=10)
-    end_param = updates_helper(value=end, default=datetime.now(timezone.utc))
-    mofo_relevant_param = updates_helper(value=mofo_relevant, default=None)
-    return get_bulk_contacts_by_timestamp(
-        db=db,
+    bulk_request = BulkRequestSchema(
         start_time=start,
-        end_time=end_param,
-        after=after_param,
-        limit=limit_param,
-        mofo_relevant=mofo_relevant_param,
+        end_time=end,
+        limit=limit,
+        after=after,
+        mofo_relevant=mofo_relevant,
     )
+    return get_bulk_contacts_by_timestamp_or_4xx(db=db, **bulk_request.dict())
```

Comment:
Converted query_params into a `BulkRequestSchema` object with `@validators` to handle the defaulting and validation logic. 

_Distributes values to `get_bulk_contacts_by_timestamp_or_4xx` in expanded fashion: `**bulk_request.dict()`_